### PR TITLE
fix: #1114 static files in `blog/assets` is not working

### PR DESCRIPTION
**Pull request by tsmrachel**
## Motivation

#1114 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

**checked that it's working for docs** - created docs/assets folder, added image to docs, and placed image in assets folder
image

**checked that it's working for blog** - created blog/assets folder, added image to blog, and placed image in assets folder

image

## Related PRs

N.A.

### CODE REVIEW

**Comment by tsmrachel** [comment]
will especially appreciate any feedback on how the tests can improved. i.e.
maybe i should move the replaceAssetsLink tests into utils.test instead?
or maybe i should create separate blog fixtures for the replaceAssetsLink in blog.test, instead of reusing the docs fixtures?

**Comment by endiliey** [comment]
Thanks @tsmrachel 😊

File: v1/lib/server/utils.js [hunk 1]
```diff
@@ -4,12 +4,12 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
 const cssnano = require('cssnano');
 const autoprefixer = require('autoprefixer');
 const postcss = require('postcss');
 const path = require('path');
 const escapeStringRegexp = require('escape-string-regexp');
+const siteConfig = require('../../website/siteConfig.js');
```
Comment (aeneasr) [inline v1/lib/server/utils.js]: This broke docusaurus completely, see #1149

Comment (JoelMarcey) [reply v1/lib/server/utils.js]: Yeah, that doesn't look right. 

@yangshun @endiliey -- I think we need to fix this or revert this specific diff and push out a hotfix.

Comment (JoelMarcey) [reply v1/lib/server/utils.js]: https://github.com/facebook/Docusaurus/pull/1150

Comment (endiliey) [reply v1/lib/server/utils.js]: Yeah this PR broke the release. My bad to overlook this. Will release a hotfix in ~1 hour.

**Pull request merged**